mapping.yaml

```diff
@@ -365,6 +365,18 @@ languages:
   iw:
     - hebrew
     - hebrew-staggered
+  kab:
+    - azerty
+    - swiss
+    - qwerty
+    - qwertz
+    - dvorak
+    - colemak
+    - colemak_dh
+    - colemak_dh_ansi
+    - bepo
+    - pcqwerty
+    - nordic
```

Comment:
You need to add `kabyle` to this list, probably at the top, else the `kabyle.yaml` layout will not be presented as an option.

Reply:
@abb128  Added kabyle to mapping.yaml